router/actions-model.js

```diff
@@ -0,0 +1,68 @@
+const db = require("../data/dbConfig");
+
+module.exports = {
+  find,
+  findById,
+  add,
+  update,
+  remove
+};
+
+function find() {
+  return db("actions")
+    .join("projects", "actions.project_id", "=", "projects.id")
+    .select(
+      { project: "projects.name" },
+      { "project description": "projects.description" },
+      { "action id": "actions.id" },
+      "actions.description",
+      "actions.notes",
+      "actions.completed",
+      "actions.created_at",
+      "actions.updated_at"
+    );
+}
+
+function findById() {
+  return db("actions")
+    .where({ "actions.project_id": id })
+    .first()
+    .join("projects", "actions.project_id", "=", "projects.id")
+    .select(
+      { project: "projects.name" },
+      { "project description": "projects.description" },
+      { "action id": "actions.id" },
+      "actions.description",
+      "actions.notes",
+      "actions.completed",
+      "actions.created_at",
+      "actions.updated_at"
+    );
+}
+
+function add(action) {
+  return db("actions")
+    .insert(action, "id")
+    .then(([id]) => {
```

Comment:
No needs to handle promises inside of this helpers functions, you can do that in your router file.